Xamarin.Forms.Platform.Android/FastRenderers/MediaElementRenderer.cs

```diff
@@ -238,6 +239,13 @@ void OnElementChanged(ElementChangedEventArgs<MediaElement> e)
 			ElementChanged?.Invoke(this, new VisualElementChangedEventArgs(e.OldElement, e.NewElement));
 		}
 
+		void MetadataRetrieved(object sender, EventArgs e)
+		{
+			((IMediaElementController)MediaElement).Duration = _view.DurationTimeSpan;
+			((IMediaElementController)MediaElement).VideoHeight = _view.VideoHeight;
+			((IMediaElementController)MediaElement).VideoWidth = _view.VideoWidth;
```

Comment:
I think that frequent unboxing of `MediaElement` should be replaced to private variable, which will do unboxing once when you `SetElement`.

[boxing and unboxing are computationally expensive process](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/types/boxing-and-unboxing#performance)

Reply:
This is casting of reference types so doesn't have the boxing overhead but I agree there should be a cleaner way of writing this